pkg/controller/issueapi/bulk_issue.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package issueapi
+
+import (
+	"net/http"
+
+	"github.com/google/exposure-notifications-verification-server/pkg/controller"
+)
+
+func (c *Controller) HandleBulkIssue() http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		ctx := r.Context()
+
+		realm := controller.RealmFromContext(ctx)
+		if realm == nil {
+			controller.MissingRealm(w, r, c.h)
+			return
+		}
+
+		hasSMSConfig, err := realm.HasSMSConfig(c.db)
```

Comment:
This check doesn't match what's in the template.

Reply:
This matches single-code upload?

https://github.com/google/exposure-notifications-verification-server/blob/c6a8d8b9297fa533f9f91300bcfd4064096e1512/pkg/controller/codes/issue.go#L42

Reply:
The template has checks for if an SMSConfig is defined, but as-written, it would be impossible for that code in the template to execute because this would error out first.

Reply:
Oops nvm. I'm wrong.

Reply:
realm.HasSMSConfig gives true/false (no error if NotFound)